<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecycleWidget</class>
 <widget class="QWidget" name="RecycleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="toggleAllButton">
       <property name="toolTip">
        <string>手工输入条码</string>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>229</width>
         <height>33</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="resetButton">
       <property name="toolTip">
        <string>手工输入条码</string>
       </property>
       <property name="icon">
        <iconset resource="itrac.qrc">
         <normaloff>:/res/edit.png</normaloff>:/res/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bcInputButton">
       <property name="toolTip">
        <string>手工输入条码</string>
       </property>
       <property name="icon">
        <iconset resource="itrac.qrc">
         <normaloff>:/res/edit.png</normaloff>:/res/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="pkgTableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="itrac.qrc">:/res/tip.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>1.扫描物品包条码，如条码破损或扫描枪损坏，请手动输入条码
2.扫描完后请对回收物品装篮</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Ui::PrimaryButton" name="recycleButton">
        <property name="text">
         <string>确认回收</string>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>pkgTableView</zorder>
  <zorder></zorder>
  <zorder>recycleButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ui::PrimaryButton</class>
   <extends>QPushButton</extends>
   <header>ui/buttons.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="itrac.qrc"/>
 </resources>
 <connections/>
</ui>
